ros/api/v1/hosts.py

```diff
@@ -257,7 +260,7 @@ def build_sort_expression(self, order_how, order_method):
             return (sort_order(PerformanceProfile.report_date),
                     asc(PerformanceProfile.system_id))
 
-        if order_method == 'group_name':
+        if order_method == 'group_name' and get_flag_value(FLAG_INVENTORY_GROUPS):
```

Comment:
@upadhyeammit shall we revert this change similar to above group_name filter?

Reply:
yeah!

Reply:
@upadhyeammit I agree we are not assigning anything to the order method, I just felt that checking should be done after feature flag checking. But if you are okay with this, I am okay as well 👍🏻